apps/condo/domains/settings/constants.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyGreaterThanPreviousMeterReadingIsEnabled:Checkbox; meta:Json; ticketSubmittingIsEnabled:Checkbox'`
+ */
+
+const MOBILE_FEATURE_CONFIGURATION = 'mobile-feature-configuration'
+const TICKET_SUBMITTING_SETTING = 'ticket-submitting-from-resident-mobile-app'
+const ONLY_PROGRESSION_METER_READINGS_SETTING = 'submit-only-progression-meter-readings'
```

Comment:
Maybe move these constants near to others?
apps/condo/domains/common/constants/featureflags.js

Reply:
Ok I will move these constants. I have never heard of such a rule, but it seems that many people follow it.